catalog/spatial/geocoding/spatial-geocoding-localsearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesLocalIndex.java

```diff
@@ -60,13 +60,14 @@ public GeoResult getLocation(final String location) {
                         population);
             }
         } catch (GeoEntryQueryException e) {
-            LOGGER.error("Error querying the local GeoNames index", e);
+            LOGGER.warn(
```

Comment:
This exception could be thrown for a number of reasons other than a missing index. Also, you've already made the exception message for a missing index more descriptive and that exception will still be logged here, so it may not be necessary to change this.